File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingOutlook/WorkingWithOutlookMessageFiles/CreateOutlookNote.php
<?php
namespace Aspose\Email\ProgrammingOutlook\WorkingWithOutlookMessageFiles;

use com\aspose\email\MapiNote as MapiNote;
use com\aspose\email\NoteColor as NoteColor;
use com\aspose\email\NoteSaveFormat as NoteSaveFormat;
class CreateOutlookNote{

    public static function run($dataDir=null){

        $note = new MapiNote();
        $note->setSubject("Blue color note");
        $note->setBody("This is a blue color note");
        $noteColor=new NoteColor();
        $note->setColor($noteColor->Blue);
        $note->setHeight(500);
        $note->setWidth(500);

        $noteSaveFormat=new NoteSaveFormat();

        $note->save($dataDir . "MapiNote.msg", $noteSaveFormat->Msg);

        print "Created outlook note successfully.".PHP_EOL;
    }


}
?>

File: Plugins/Aspose-Email-Java-for-Jython/asposeemail/ProgrammingOutlook/CreateOutlookNote.py
from asposeemail import Settings
from com.aspose.email import MapiNote
from com.aspose.email import NoteColor
from com.aspose.email import NoteSaveFormat

class CreateOutlookNote:

    def __init__(self):
                        
        dataDir = Settings.dataDir + 'ProgrammingOutlook/WorkingWithOutlookMessageFiles/CreateOutlookNote/'
        
        note = MapiNote()
        note.setSubject("Blue color note")
        note.setBody("This is a blue color note")
        noteColor = NoteColor
        note.setColor(noteColor.Blue)
        note.setHeight(500)
        note.setWidth(500)

        noteSaveFormat=NoteSaveFormat

        note.save(dataDir + "MapiNote.msg", noteSaveFormat.Msg)

        print "Created outlook note successfully."
        
 
if __name__ == '__main__':        
    CreateOutlookNote()